packages/@okta/vuepress-site/docs/reference/registration-hook/index.md

```diff
@@ -202,26 +202,26 @@ For the registration inline hook, the `error` object provides a way of displayin
 
 > **Note:** Generic Progressive Enrollment error messages are only available in the Identity Engine.
 
-* If you're using the Okta Sign-In Widget for Profile Enrollment, only the `errorSummary` messages of the `errorCauses` objects that your external service returns appear as inline errors, given the following:
+If you're using the Okta Sign-In Widget for Profile Enrollment, only the `errorSummary` messages of the `errorCauses` objects that your external service returns appear as inline errors, given the following:
 
-   * You don't customize the error handling behavior of the widget.
-   * The `location` of `errorSummary` in the `errorCauses` object specifies the request object's user profile attribute.
+* You don't customize the error handling behavior of the Widget.
```

Comment:
```suggestion
* You don't customize the error handling behavior of the widget.
```

Reply:
this should stay lowercase. Was Acrolinx suggesting we capitalize?

Reply:
I honestly don't remember. I wouldn't have changed anything if it wasn't though, so probably.